Apps/Src/ReadTritium.c

```diff
@@ -29,19 +33,9 @@ tritium_error_code_t MotorController_getTritiumError(void){
 	return T_NONE;
 }
 
-/**
- * @brief   Assert Error if Tritium sends error. When Fault Bitmap is set,
- *          and semaphore is posted, Fault state will run.
- * @param   motor_err Bitmap which has motor error codes
- */
-static void assertTritiumError(uint16_t motor_err){    
-	OS_ERR err;
-	if(motor_err != T_NONE){
-		FaultBitmap |= FAULT_TRITIUM;
-		OSSemPost(&FaultState_Sem4, OS_OPT_POST_1, &err);
-		assertOSError(0, err);
-	}
-}
+
+
+
 
 
 /* OBJECTIVES:
```

Comment:
Why is this `Objectives` section still here? Probably out of the scope of this particular PR, just curious as to why we're keeping it around (if it's on purpose that is)

Reply:
It's getting removed in the SendCarCAN pr